helm/resource_helm_release_test.go

```diff
@@ -592,6 +592,120 @@ func TestAccResourceRelease_updateAfterFail(t *testing.T) {
 	})
 }
 
+func TestAccResourceRelease_upgradeInstall_coldstart(t *testing.T) {
```

Comment:
In the SDKv2 codebase this feature has an additional test: https://github.com/hashicorp/terraform-provider-helm/commit/723cb76045ec2b38adee1ee3998fcdf36327169c#diff-0c630859c0a0648a5a7cd178d9b8d011108f16648cd8fc6b651390cec95ebc56R151